tests/test_grid.py

```diff
@@ -48,12 +48,12 @@
 @pytest.mark.parametrize("ggor", ["jigsaw", "gmsh"])
 def test_answer(tmpdir, window, coast, ggor):
 
-    df = pg.grid(
+    df = pmesh.set(
```

Comment:
Not related to the PR, but in other tests we name the object returned by `pmesh.set` as `mesh` or `grid`. here it is named as `df`. having more consistency is not a bad idea, but let's not block this over this.